### PR TITLE
IN-195 support UTF characters

### PR DESCRIPTION
**Pull request by sturchaninov**
changes: _rev_

### Proposed Changes

* Add support of unicode characters

### Checklist

- [x] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation (if appropriate)
- [x] I have verified that my changes have not introduced new lint errors
- [x] I have updated the change log

### CODE REVIEW

**Comment by sturchaninov** [comment]
@nzack please review

File: lib/binary.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * Provides a binary type that can be initialized with many different forms of
  * data
  */
-
+require('Buffer');
```
Comment (nzack) [inline lib/binary.js]: No need to include the `Buffer` dependency here.  It's automatically supported from NodeJS (when running tests) and is automatically pulled in by [webpack from the browser ](https://webpack.js.org/configuration/node/)

Comment (nzack) [reply lib/binary.js]: take a look at this compare to see what I mean here: https://github.com/sturchaninov/binary.js/compare/bug/IN-195-fix-unicode...virtru:IN-195-with-buffer

Comment (sturchaninov) [reply lib/binary.js]: @nzack cool. Will you merge your changes? Is I need to do anything except updating `binary` version in `sss-engine` and `bp` ?

Comment (nzack) [reply lib/binary.js]: Feel free to merge/incorporate the changes yourself

**Comment by nzack** [comment]
Merging at Sergii's request.

**Pull request merged**